package.json

```diff
@@ -38,7 +38,7 @@
     "squid:codegen": "yarn workspace multix-squid codegen",
     "squid:typegen": "yarn workspace multix-squid typegen",
     "squid:build": "yarn workspace multix-squid build",
-    "squid:install": "yarn workspaces focus multix-squid",
+    "squid:install": "yarn workspace multix-squid install",
```

Comment:
same here, the focus makes sure that only the needed depandancies are installed.

With the current dockerfile, we install twice everything.